cleverhans/utils.py

```diff
@@ -220,6 +220,97 @@ def grid_visual(data):
     return figure
 
 
+def get_logits_over_interval(sess, model, x, adv_x, x_data,
+                             min_epsilon=-10, max_epsilon=10,
+                             num_points=21):
+    """Get logits when the input is perturbed in an interval in adv direction.
+
+    Args:
+        sess: Tf session
+        model: Model for which we wish to get logits
+        x: Data tensor
```

Comment:
I realized I'm confused about the interface here.
Is x meant to be a minibatch or a single example? I would expect it to be a single example.
Is the user required to make it be a minibatch of size 1?

Reply:
I have updated the code so that x is now supposed to be of shape [height, width, channels]